frontend/src/main/components/Announcement/AnnouncementForm.js

```diff
@@ -56,27 +53,28 @@ function AnnouncementForm({ initialContents, submitAction, buttonLabel = "Create
                     isInvalid={Boolean(errors.startDate)}
                     {...register("startDate", {
                         required: "StartDate is required.",
-                        pattern: isodate_regex
                     })}
                 />
                 <Form.Control.Feedback type="invalid">
                     {errors.startDate && 'Start Date is required and must be provided in ISO format.'}
                 </Form.Control.Feedback>
             </Form.Group>
 
-            <Form.Group className="mb-3" >
+            <Form.Group className="mb-3">
                 <Form.Label htmlFor="endDate">End Date</Form.Label>
                 <Form.Control
                 // Stryker disable next-line all
                     data-testid={testIdPrefix + "-endDate"}
                     id="endDate"
                     type="datetime-local"
-                    isInvalid={Boolean(errors.endDate)}
-                    // Stryker disable next-line all
+                    isInvalid={Boolean(errors.startDate)}
                     {...register("endDate", {
-                        pattern: isodate_regex
+                        required: "endDate is required.",
                     })}
                 />
+                <Form.Control.Feedback type="invalid">
+                    {errors.endDate && 'End Date is required and must be provided in ISO format.'}
+                </Form.Control.Feedback>
             </Form.Group>
```

Comment:
Why are we making end date required?

It was a feature, not a bug that it was optional.  That allows you to put in an announcement that stays up indefinitely.

Reply:
I thought that It would prevent mistakes as we don't have a delete announcement or edit announcement functionality yet on the frontend. 

I added the change - end date is no longer required (and updated the tests)